components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -153,12 +153,22 @@ const UpdateOrderPopUp = ({ setMenuState, contribution, createNotification, setS
   }, [mappedTierOptions]);
 
   const getTierOptions = () => {
+    let optionObject;
+    const objectArray = [];
+    const flexible = selectedTier.flexible || selectedTier.value?.flexible;
```

Comment:
:ice_cream: Having to check `selectedTier.___ || selectedTier.value?.___` each time adds a bit of complexity, would be nice to be consistent on the type stored there.